web-client/src/presenter/actions/AdvancedSearch/getPractitionersByNameAction.ts

```diff
@@ -1,3 +1,4 @@
+import { ALL_SELECTION } from '@shared/business/entities/cases/CaseSearch';
```

Comment:
This constant feels like it is in a funny place now (we are searching for practitioners, not cases), but I'm not sure exactly where else to put it at the moment.